pkg/scannerv4/client/client.go

```diff
@@ -184,31 +183,19 @@ func createGRPCConn(ctx context.Context, o connOptions) (*grpc.ClientConn, error
 func (c *gRPCScanner) GetSBOM(ctx context.Context, ref name.Digest) ([]byte, bool, error) {
 	// verify index report exists for the image
 	hashId := getImageManifestID(ref)
-	_, found, err := c.GetImageIndex(ctx, hashId)
+	ir, found, err := c.GetImageIndex(ctx, hashId)
 	if err != nil {
 		return nil, false, err
 	}
 	if !found {
 		return nil, false, nil
 	}
 
-	// for testing only
-	sbom := map[string]interface{}{
-		"SPDXID":      "SPDXRef-DOCUMENT",
-		"spdxVersion": "SPDX-2.3",
-		"creationInfo": map[string]interface{}{
-			"created": "2023-08-30T04:40:16Z",
-			"creators": []string{
-				"Organization: Xyz",
-				"Tool: FOSSA v0.12.0",
-			},
-		},
-	}
-	sbomBytes, err := json.Marshal(sbom)
-	if err != nil {
-		return nil, false, err
-	}
-	return sbomBytes, true, nil
+	resp, err := c.matcher.GetSBOM(ctx, &v4.GetSBOMRequest{
+		HashId:   ir.GetHashId(),
```

Comment:
We might not even need `HashId`, if the SBOM bytes don't care about the "image reference".

Reply:
We will be adding the digest to the SBOM so that the image it references can be uniquely identified.